sound/soc/sof/sof-pci-dev.c

```diff
@@ -291,6 +292,8 @@ static void sof_pci_remove(struct pci_dev *pci)
 	/* release firmware */
 	release_firmware(sof_pdata->fw);
 
+	pm_runtime_get_noresume(&pci->dev);
+
```

Comment:
@plbossart I followed the skl driver and also one of the comments in the pci-driver.c that says:
/*
         * Unbound PCI devices are always put in D0, regardless of
         * runtime PM status.  During probe, the device is set to
         * active and the usage count is incremented.  If the driver
         * supports runtime PM, it should call pm_runtime_put_noidle(),
         * or any other runtime PM helper function decrementing the usage
         * count, in its probe routine and pm_runtime_get_noresume() in
         * its remove routine.
         */

I can check with Rafael to understand this a bit more and get back.

Reply:
good info, thanks. Please add a comment that this follows recommendations in pci-driver.c for both cases so that we can track why it was done this way.